package.json

```diff
@@ -44,21 +44,21 @@
         "test:watch": "ts-scripts test --watch asset --"
     },
     "devDependencies": {
-        "@terascope/data-types": "~1.7.3",
-        "@terascope/elasticsearch-api": "~4.8.1",
-        "@terascope/elasticsearch-asset-apis": "~1.2.0",
-        "@terascope/eslint-config": "~1.1.5",
-        "@terascope/job-components": "~1.9.3",
-        "@terascope/scripts": "~1.10.1",
-        "@terascope/teraslice-state-storage": "~1.8.1",
+        "@terascope/data-types": "~1.7.4",
+        "@terascope/elasticsearch-api": "~4.8.2",
+        "@terascope/elasticsearch-asset-apis": "~1.2.1",
+        "@terascope/eslint-config": "~1.1.6",
```

Comment:
Maybe if we excluded this dependency change it would work.

Reply:
The tests failed because of a lint error, I fixed the lint error, but why are we running lint on every test?

Reply:
I was just pondering the same question, would make sense to pull that out in front of everything else.

Reply:
Realistically, we should make that a git pre-push hook so we don't even burn up CI time on stuff that fails linting.